core/src/test/java/org/elasticsearch/index/query/CommonTermsQueryBuilderTests.java

```diff
@@ -173,4 +190,24 @@ public void testCommonTermsQuery4() throws IOException {
         ExtendedCommonTermsQuery ectQuery = (ExtendedCommonTermsQuery) parsedQuery;
         assertThat(ectQuery.isCoordDisabled(), equalTo(disableCoord));
     }
+
+    public void testParseFailsWithMultipleFields() throws IOException {
+        String json = "{\n" +
+                "  \"common\" : {\n" +
+                "    \"message1\" : {\n" +
+                "      \"query\" : \"nelly the elephant not as a cartoon\"\n" +
+                "    },\n" +
+                "    \"message2\" : {\n" +
+                "      \"query\" : \"nelly the elephant not as a cartoon\"\n" +
+                "    }\n" +
+                "  }\n" +
+                "}";
+
+        try {
+            parseQuery(json);
+            fail("parseQuery should have failed");
+        } catch(ParsingException e) {
+            assertEquals("[common] query doesn't support multiple fields, found [message1] and [message2]", e.getMessage());
+        }
```

Comment:
I pushed a commit that introduces using expectThrows wherever possible in our query tests. That way I will remember it for the next time as I had to introduce a kagillion  of those... also PR stats got much better :)